test/evp_extra_test.c

```diff
@@ -38,6 +38,10 @@
 #include "crypto/evp.h"
 #include "fake_rsaprov.h"
 
+#ifdef STATIC_LEGACY
+OSSL_provider_init_fn ossl_legacy_provider_init;
```

Comment:
Is this an extern ? (Looks like we do this elsewhere also)

Reply:
Yes, it is. But we do not declare it in any header as this is really unneeded outside of these test code hacks.